### PR TITLE
oversteer: init at 0.7.2

### PR DESCRIPTION
**Pull request by srounce**
###### Description of changes

Add Oversteer - Steering Wheel Manager for Linux at version [0.7.2](https://github.com/berarma/oversteer/releases/tag/0.7.2).

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by JulienMalka — CHANGES_REQUESTED** [review]
Hello, thanks for your contribution !
Could you add yourself to the maintainer list : ``maintainers/maintainer-list.nix``, in another commit.

**Comment by srounce** [comment]
Found there's a problem with the udev rules that this generates, will try and fix and update

**Comment by srounce** [comment]
Patched the udev rules to use bash from nixpkgs.

**Comment by srounce** [comment]
Tested with Logitech G29 and works as it should (with FFB support when using lg4ff) driver.

File: pkgs/applications/misc/oversteer/default.nix [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    with p; [
+      pygobject3
+      pyudev
+      pyxdg
+      evdev
+      matplotlib
+      scipy
+      gtk3
+      pygobject3
+    ]));
```
Comment (zimbatm) [inline pkgs/applications/misc/oversteer/default.nix:20]:
```suggestion
  python = python3.withPackages (p:
    with p; [
      pygobject3
      pyudev
      pyxdg
      evdev
      matplotlib
      scipy
      gtk3
      pygobject3
    ]);
```

**Comment by zimbatm** [comment]
Result of `nixpkgs-review pr 211423` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>oversteer</li>
  </ul>
</details>

File: pkgs/applications/misc/oversteer/default.nix (hunk 1 above)
Comment (srounce) [reply pkgs/applications/misc/oversteer/default.nix]: Thx, fixed :+1:

**Review by zimbatm — APPROVED** [review]

**Comment by Toomoch** [comment]
As far as I can see, this package doesn't add the udev rules automatically with `services.udev.packages`, right?

**Comment by zimbatm** [comment]
@srounce ^

**Pull request merged**